### ISSUE TITLE
Minor inconsistency: DoFs vs dofs

### ISSUE THREAD

**Issue by lipnikov** [amanzi/amanzi#393]
A minor inconsistency in the name "DoFs". Amanzi base code uses lower-case "dofs" everywhere, except for one line:

   src/state/independent_variable_field_evaluator_fromfile.cc
       ndofs_ = plist.get<int>("number of DoFs", 1);

**Comment by ecoon** [comment]
@lipnikov  Should this file also change on master?  

`./doc/input_spec/AmanziNativeSpecV8.rst`

Or is there a "dev" or "in progress for next version" doc file somewhere that I'm missing?

**Comment by lipnikov** [comment]
Yes. SpecV8 has two lines with the camel-case DoFs.  Since this is the development spec, there is no pressing needs to keep a "dev" version.

**Comment by lipnikov** [comment]
Yes. SpecV8 has two lines with the camel-case DoFs.  Since this is the development spec, there is no pressing needs to keep a "dev" version.
